src/client/game.cpp

```diff
@@ -2190,8 +2190,11 @@ void Game::processItemSelection(u16 *new_playeritem)
 
 	/* Item selection using mouse wheel
 	 */
+	Inventory *inventory = &player->inventory;
+	InventoryList *mainlist = inventory->getList("main");
+	s32 mainlist_size = (mainlist == nullptr) ? PLAYER_INVENTORY_SIZE : mainlist->getSize();
```

Comment:
Is it correct to fall back to `PLAYER_INVENTORY_SIZE` if the main list is undefined / not yet sent?

See also: https://github.com/minetest/minetest/pull/11926#discussion_r790147379

Reply:
If `mainlist` is not defined I would simply not process hotbar changes

Reply:
Fixed in 10e8aab1c